src/browser.ts

```diff
@@ -278,7 +290,9 @@ export class AnalyticsBrowser {
   static standalone(
     writeKey: string,
     options?: InitOptions
-  ): Promise<Analytics> {
-    return AnalyticsBrowser.load({ writeKey }, options).then((res) => res[0])
+  ): Promise<StandaloneAnalytics> {
```

Comment:
I don't think the return type of this should be changed. If I recall, this function returns an instance of `Analytics`, not `StandaloneAnalytics`. The main difference being that `StandaloneAnalytics` is an iterable.